doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,289 @@
+         $ cd acrn-kernel
+
+   b. Copy kernel_config_uefi_sos to .config
+   c. make olddefconfig
+   d. make menuconfig
+
+      .. figure:: images/acrn_qemu_4.png
+         :align: center
+
+   f. make
+   g. Copy ``arch/x86/boot/bzImage`` to the Service VM guest /boot/ directory
+
+   .. note::
+      Since the needed drivers are part of the Service VM kernel itself, we can avoid installing the modules.
+
+5. Update Ubuntu GRUB to boot ACRN hypervisor and load ACRN Kernel Image. Append the following
+   configuration to the :file:`/etc/grub.d/40_custom`,
+
+   .. code-block:: none
+
+      menuentry 'ACRN hypervisor' --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
+         recordfail
+         load_video
+         gfxmode $linux_gfx_mode
+         insmod gzio
+         insmod part_msdos
+         insmod ext2
+
+         echo 'Loading ACRN hypervisor with SDC scenario ...'
+         multiboot --quirk-modules-after-kernel /boot/acrn.32.out
+         module /boot/bzImage_sos Linux_bzImage
+      }
+
+6. Update GRUB ``sudo update-grub``.
+
+7. Shutdown the guest and relaunch using, ``virsh start ACRNSOS --console`` and select ACRN hypervisor from GRUB menu to launch Service VM running on top of ACRN.
+   This can be verified from ``dmesg`` as shown below,
+
+   .. code-block:: none
+
+      guestl1@ACRNSOS:~$ dmesg | grep ACRN
+      [    0.000000] Hypervisor detected: ACRN
+      [    2.337176] ACRNTrace: Initialized acrn trace module with 4 cpu
+      [    2.368358] ACRN HVLog: Initialized hvlog module with 4 cpu
+      [    2.727905] systemd[1]: Set hostname to <ACRNSOS>.
+
+8. When shutting down, make sure to cleanly destroy the Service VM to prevent crashes in subsequent boots. This can be done using,
+
+      .. code-block:: none
+
+         virsh destory ACRNSOS # where ACRNSOS is the virsh domain name.
+
+
+Service VM Networking updates for User VM
+*****************************************
+If you are planning to enable networking for User VM(L2 guest), the following steps might be helpful.
+
+1. Edit your :file:`/etc/netplan/01-netcfg.yaml` file to add acrn-br0 as below,
+
+   .. code-block:: none
+
+      network:
+         version: 2
+         renderer: networkd
+         ethernets:
+            enp1s0:
+               dhcp4: no
+         bridges:
+            acrn-br0:
+               interfaces: [enp1s0]
+               dhcp4: true
+               dhcp6: no
+
+2. Apply the new network configuration by,
+
+      .. code-block:: none
+
+         $ cd /etc/netplan
+         $ sudo netplan generate
+         $ sudo netplan apply
+
+3. Create a tap interface (tap0) and add the tap interface as part of the acrn-br0.
+   Copy the following files to :file:`/usr/lib/system/network`,
+
+   - `50-acrn.network <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/misc/acrnbridge/acrn.network>`__
```

Comment:
We can instruct the user to pick up these files from the local copy of the repo, the added benefit is that it will automatically be in sync with the version of ACRN that was checked out and built.